nncf/experimental/openvino_native/quantization/quantize.py

```diff
@@ -83,18 +87,23 @@ def quantize_with_accuracy_control(model: ov.Model,
                                     fast_bias_correction, model_type, ignored_scope, compress_weights=False)
 
     # Backends
-    backend = get_backend(model)
-    algo_backend = get_algo_backend(backend)
+    if algo_backend is None:
+        backend = get_backend(model)
+        algo_backend = get_algo_backend(backend)
 
+    nncf_logger.info('Validation of initial model was started')
     initial_metric = validation_fn(algo_backend.prepare_for_inference(model),
                                    validation_dataset.get_data())
     nncf_logger.info(f'Metric of initial model: {initial_metric}')
 
+    nncf_logger.info('Validation of quantized model was started')
     quantized_metric = validation_fn(algo_backend.prepare_for_inference(quantized_model),
                                      validation_dataset.get_data())
     nncf_logger.info(f'Metric of quantized model: {quantized_metric}')
 
-    accuracy_aware_loop = QuantizationAccuracyRestorer(algo_backend, max_drop=max_drop)
+    accuracy_aware_loop = QuantizationAccuracyRestorer(algo_backend,
```

Comment:
It looks like, the question had to be addressed here https://github.com/openvinotoolkit/nncf/pull/1553 but I follow up in this PR.

Can you refactor code so that the `algo_backend` is removed from the input argument list of the `init` method of `QuantizationAccuracyRestorer?`

Reply:
Done